Functions/src/main/java/aws/devopsguru/partner/servicenow/ServiceNowConnector.java

```diff
@@ -187,6 +193,17 @@ private Incident updateIncident(Context context, Incident incident) {
 
     // An HttpRequest instance is built through an HttpRequest builder
     private HttpRequest.Builder getBuilderWithBasicValues(String uriPath) {
+
+        JSONObject serviceNowCredentials = getSecret(secretName);
+
+        String username = new String();
```

Comment:
Only declare the variables.  ```String userName```. In Java, there is no need to create empty String Objects.